### DISCUSSION TITLE
2006/07/21/%E5%B7%A5%E4%BD%9C%E4%B9%8B%E4%BD%99%EF%BC%8C%E7%8E%A9%E4%B8%AA%E5%B0%8F%E6%B8%B8%E6%88%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#96] [Announcements]
# 2006/07/21/%E5%B7%A5%E4%BD%9C%E4%B9%8B%E4%BD%99%EF%BC%8C%E7%8E%A9%E4%B8%AA%E5%B0%8F%E6%B8%B8%E6%88%8F

这个游戏非常简单，是 Windows 自带的，用来测试你点击鼠标的速度。

https://blog.qizhen.xyz/2006/07/21/%E5%B7%A5%E4%BD%9C%E4%B9%8B%E4%BD%99%EF%BC%8C%E7%8E%A9%E4%B8%AA%E5%B0%8F%E6%B8%B8%E6%88%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


fx
2006年08月15日 18:25 编辑
差点重新启动了

回复
zhang
2006年07月30日 15:22 编辑
弹出好多窗口，不好玩

回复
kuo
2006年07月23日 19:41 编辑
你是谁呀？

回复
Artex (Han Xiao)
2006年07月22日 07:46 编辑
呵呵，太幽默了，的确被吓倒了。:)
欢迎来我的Space!

回复
yi
2006年07月21日 21:37 编辑
果然。。。弹出9个。。。

回复
小桔
2006年07月21日 13:30 编辑
为什么来看我的空间呢？为什么不留言呢？好纳闷。

回复
小桔
2006年07月21日 13:30 编辑
为什么来看我的空间呢？为什么不留言呢？好纳闷。
